internal/store/postgres/migrations/000056_capitalize_secret_name.up.sql

```diff
@@ -0,0 +1 @@
+UPDATE SECRET SET NAME = UPPER(NAME);
```

Comment:
Add a comment in down file to explain why the file is empty.

Reply:
sure